stories/tabs.stories.mdx

```diff
@@ -0,0 +1,46 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Next Version Implementation Guides/Components/Navigation/Tabs" />
+
+
+## Implementation guidelines for next tabs component 
+
+It should use text basic  component to display the labels 
+The latest tabs should display the separation line at the end to mark the end of the tab
+
+Expected component API : 
+
+```
+<Tabs variant={backgroundLevel3|backgroundLevel4}>
+    <Tab path={string} label={} textBadge={} >{children}</Tab>
+    <Tab path={string} label={} textBadge={} >{children}</Tab>
+    <Tab path={string} label={} textBadge={} >{children}</Tab>
+</Tabs>
+```
+
+Tabs are based on react router and extends the current route architecture of the application with child routes.
+Each path is taken as a child path to the current router path.
+
+Tab size depend on the label, minimum size is 5rem.
```

Comment:
Tab **height** is fixed, it's the tab **width** that is dependent on the label string length.